lib/src/signed_video_h26x_auth.c

```diff
@@ -759,11 +758,13 @@ maybe_validate_gop(signed_video_t *self, h26x_nalu_t *nalu)
       if (item->nalu && item->nalu->is_gop_sei && item->validation_status == 'P') {
         const uint8_t *tlv_data = item->nalu->tlv_data;
         size_t tlv_size = item->nalu->tlv_size;
-        status = tlv_find_tag_and_decode(self, tlv_data, tlv_size, PUBLIC_KEY_TAG, false);
-        if (status == SVI_OK) {
+        svi_rc ret = SVI_UNKNOWN;
```

Comment:
The name `ret` indicates that it is a variable that is returned. I suggest removing `ret` entirely; See comment below.